### DISCUSSION TITLE
test/production; exports is not defined

### DISCUSSION THREAD

**Discussion by jfuehner** [vuejs/vue-cli#7088] [Q&A]
Hi All!

Hoping for a little help. I just recently switched to using vue-cli 5.0.3 with webpack v5.

I have a web app that is throwing a “exports is not defined” error from within one of my node dependencies.

The interesting thing is the error only occurs when I build with NODE_ENV set to test or production. All works fine with development builds; no errors.

Any idea how I can fix this?

**Comment by Viggo95** [comment]
The information you gave is too little to make a judgment. suggest a mini repo
